src/services/syncer/strategies/transifex/utils/transformer.js

```diff
@@ -212,6 +211,17 @@ function parseSourceStringForAPI(key, payload) {
     result.occurrences = _.join(payload.meta.occurrences, ',');
   }
 
+  const context = _.get(payload, 'meta.context');
+  if (context) {
+    if (Array.isArray(context)) {
+      result.context = _.join(context, ':');
+    } else {
+      result.context = context;
+    }
+  } else {
+    result.context = '';
+  }
+
```

Comment:
_Suggestion:_ I would probably simplify this to:
```
const context = _.get(payload, 'meta.context') || '';
if (Array.isArray(context)) {
  result.context = _.join(context, ':');
} else {
  result.context = context;
}
```

Reply:
> @codegaze @kostasgeo Maybe you should also update README to reflect the change that from now on the "official" way to pass the context on the POST /content URL is a string instead of an array? Then we can gradually migrate the SDKs to reflect the change there as well.

@nbasili We have actually decided to leave the API that CDS exposes as is and we internally modify the list to a string joined with `:`. However, since the validation is not strict, it already works with strings too. Do you think we should still change the README?

Reply:
@kostasgeo I think yes, this way, the array mode is a "deprecated" feature and the new way is to pass it as string. New SDKs will need to use the "string" and old SDKs will be migrated to string in new releases. Otherwise we will leave a tech-debt there. In any case the README is strictly informative. The fix is great!